cmd/skipper/defaultfilterflags.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/zalando/skipper/eskip"
+)
+
+const (
+	defaultPrependFiltersUsage = "set of default filters to apply to prepend to all filters of all routes"
+	defaultAppendFiltersUsage  = "set of default filters to apply to append to all filters of all routes"
+)
+
+type defaultFiltersFlags struct {
+	filters []*eskip.Filter
+}
+
+func (dpf *defaultFiltersFlags) String() string {
+	res := make([]string, len(dpf.filters))
+	for _, f := range dpf.filters {
+		res = append(res, f.Name)
```

Comment:
this will print sg like: `foo -> bar -> baz`. Where is it used?

Reply:
only if you print the flags, so I don't really care